### PR TITLE
JavaScript refactored to classes

### PR DESCRIPTION
**Pull request by mariegrace31**
In this milestone, my partner and I,
- Added a class to manage the books
- Added buttons to add and remove the books
- Added a method to update the stored books in local storage

### CODE REVIEW

**Review by AbelG101 — COMMENTED** [review]
# Hi @mariegrace31 & @Peter-Kibet 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- You have submitted a project, that doesn't have all the proper sections of the Microverse readme template. I noticed you removed the "Future features" & the "License" section from your readme please take note that these sections should always be included with the proper content accordingly.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: style.css [hunk 1]
```diff
@@ -1,25 +1,84 @@
...
+    margin-left: 190px;
+  }
+
+  input {
+    width: 65%;
+  }
+
+  #books ul {
+    width: 90%;
+  }
```
Comment (divinecharlotte) [inline style.css]: - [ ] kindly consider adding all your designs in the center to meet the design requirements on the wireframe cause currently its on the left

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Team       , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Use of GitHub flow👍 
- Descriptive pr 👍 

 




 

## Required Changes ♻️

- [ ] Kindly consider adding a working link to your issue page under the contribution section to give a good user experience to the person who would like to interact with it 
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css
Comment (KanzaTahreem) [inline style.css:1]: - [OPTIONAL] Good job with the design 🥇 I noticed a bit of horizontal scrolling. Can you please fix this for providing a good user experience? 😃 You can consider adding the following or simply `margin: 0` under `body` selector 👇

```
* {
    box-sizing: border-box;
    margin: 0;
    padding: 0;
  }
```

File: README.md [hunk 2]
```diff
@@ -62,7 +67,7 @@ Install this project with: cd awesomebooks-project npm install
 
 ## 🤝 Contributing <a name="contributing"></a>
 
-Contributions, issues, and feature requests are welcome! And here is the link to the "issues" section:https://github.com/mariegrace31/awesomebooks-project/issues
+Contributions, issues, and feature requests are welcome! And here is the link to the "issues" section: https://github.com/mariegrace31/awesomebooks-project/issues
```
Comment (KanzaTahreem) [inline README.md:70]: - [OPTIONAL] The issues link is working 🥇 How about using something like below for providing issues page link to increase readability 💪✨

  > _Contributions, issues, and feature requests are welcome! Feel free to check the [issues page](https://github.com/mariegrace31/awesomebooks-project/issues)_

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @mariegrace31 & @Peter-Kibet  🙋‍♀️

Great work implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please see the comments below 👇_

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**